can_bus/quadruna/BMS/BMS_alerts.json

```diff
@@ -35,17 +35,17 @@
         "RxOverflow" : {
             "id" :108,
             "description" : "Example"
+        },
+        "MissingRSMHeartbeat":{
+            "id" :152,
+            "description" : "Example"
         }
     },
     "faults": {
         "MissingVCHeartbeat":{
             "id" :151,
             "description" : "Example"
         },
```

Comment:
Doesn't look like the BMS needs to check the VC or the RSM heartbeat. It doesn't use any signals from either board anywhere, so I think we can just delete these checks. What do you think? @liam-ilan @Lucien950